autoPyTorch/utils/implementations.py

```diff
@@ -59,3 +63,123 @@ def __call__(self, y: Union[np.ndarray, torch.Tensor]) -> np.ndarray:
     @staticmethod
     def get_properties() -> Dict[str, Any]:
         return {'supported_losses': ['BCEWithLogitsLoss']}
+
+
+class MinorityCoalesceTransformer(BaseEstimator, TransformerMixin):
+    """ Group together categories whose occurrence is less than a specified min_frac."""
+    def __init__(self, min_frac: Optional[float] = None):
+        self.min_frac = min_frac
+        self._categories_to_coalesce: Optional[List[np.ndarray]] = None
+
+        if self.min_frac is not None and (self.min_frac < 0 or self.min_frac > 1):
+            raise ValueError(f"min_frac for {self.__class__.__name__} must be in [0, 1], but got {min_frac}")
+
+    def _check_dataset(self, X: Union[np.ndarray, sparse.csr_matrix]) -> None:
+        """
+        When transforming datasets, we modify values to:
+            * -1 for missing values
```

Comment:
isnt it 0 for missing values?

Reply:
So I think you need to update the documentation. I dont think it should affect the functionality.